test/endtoend/endToEndSos.test.ts

```diff
@@ -30,7 +30,7 @@ const nextConfig = {
     watcherResponseConfirmations: 0,
     maximumReorgLimit: 10
 };
-const provider = new ethers.providers.Web3Provider(ganache);
+const provider = new ethers.providers.Web3Provider(ganache as ethers.providers.AsyncSendable);
```

Comment:
You can do the cast once where ganache is declared.

Reply:
You're right, much cleaner. Done in dde7c9f.